### DISCUSSION TITLE
How do I display all results in a cell?

### DISCUSSION THREAD

**Discussion by saiwing-yeung** [marimo-team/marimo#3058] [Q&A]
In Jupyter, if I do
```
from IPython.core.interactiveshell import InteractiveShell
InteractiveShell.ast_node_interactivity = 'all'

x, y = 1, 2
x
y
```
x and y will be displayed in separate lines. How do I do the same in marimo? I know I can `print()` but they are not the same. Thanks!

**Comment by mscolnick** [comment]
you can use `mo.output.append(result)` or `mo.output.replace(result)`. https://docs.marimo.io/api/outputs.html#outputs
